src/service/receipt.service.ts

```diff
@@ -0,0 +1,130 @@
+
+import chalk from 'chalk';
+import {
+    ArtifactExpiryReceipt,
+    BalanceChangeReceipt,
+    BalanceTransferReceipt,
+    InflationReceipt,
+} from 'nem2-sdk';
+
+export class ReceiptService {
+    constructor() {
+
+    }
+
+    /**
+     * @description The logic of formatting Receipt info
+     * @param receipt
+     * @returns {string}
+     */
+    public formatReceiptToFilter(statement: any): string {
+        let txt = '';
+
+        // render transactionStatements
+        txt += chalk.green('transactionStatements:\t') + '\n';
+        txt += '-'.repeat('transactionStatements:\t'.length) + '\n\n';
+        for (let transactionIdx = 0; transactionIdx < statement.transactionStatements.length; transactionIdx++) {
+            txt += 'hight:\t\t' + statement.transactionStatements[transactionIdx].height + '\n';
+            for (let receiptIdx = 0; receiptIdx < statement.transactionStatements[transactionIdx].receipts.length; receiptIdx++) {
+                txt += '<index: ' + transactionIdx + '-' + receiptIdx + '>\t';
+                const receiptItem = statement.transactionStatements[0].receipts[0];
```

Comment:
Probably an issue. The output will only render the first transaction statement [0] and first receipt [0].